_layouts/index.html

```diff
@@ -17,7 +17,14 @@
       {% if signature.affiliation %}
       , {{ signature.affiliation }}
       {% endif %}
+
+      {% if signature.title %}
+      , {{ signature.title }}
+      {% endif %}
     </li>
+    {% if signature.github %}
+    <!-- Github user: {{ signature.github }} -->
+    {% endif %}
```

Comment:
no, I've really got to get back to work, but maybe make an issue later? It'll be an easy fix once the PR is already merged.